api/conf/conf.yaml

```diff
@@ -14,12 +14,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-version: '3'
+# PLEASE DO NOT UPDATE THIS FILE!
```

Comment:
it makes me confused, please confirm we need those warning message

Reply:
it is useless. i will remove them